### ISSUE TITLE
Incompatible with Apotheosis

### ISSUE THREAD

**Issue by rtxyd** [Xaikii/ArmorAdjustment#1]
1.19.2 Modpack with my friends. Mod version 1.2.0
It's normal at first, about 1 minute, then crash, and I will get crash again when I enter this world.

[crash-2023-03-23_04.19.53-server.txt](https://github.com/Xaikii/ArmorAdjustment/files/11044467/crash-2023-03-23_04.19.53-server.txt)

**Comment by rtxyd** [comment]
after deleting this mod, I can enter the save.

**Comment by Xaikii** [comment]
Yes, as I state on the CurseForge Page, it's Incompatible with Mods that also use Mixins at the same Function.
This is the case for Apotheosis.

**Comment by rtxyd** [comment]
Would you consider to fix this?

**Comment by Xaikii** [comment]
The message is there for a reason...
But if you can tell me how I can make it not crash when another Mod also uses a Mixin there, sure.
Until then, no!
